src/Orleans/Configuration/Options/StatisticsOptions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using System;
+
+namespace Orleans.Configuration
```

Comment:
Should be `Orleans.Hosting` I think

Reply:
If we decide to put all options to hosting, sure, move them there!